File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/Delphi/IssueList/IssueListSchedule.pas
unit IssueListSchedule;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, IssueListForm, cxDropDownEdit, cxCalendar, cxDBEdit,
  cxImageComboBox, cxMaskEdit, cxLookupEdit, cxDBLookupComboBox, cxControls,
  cxContainer, cxEdit, cxTextEdit, ExtCtrls, StdCtrls, cxSpinEdit, DBCtrls,
  cxGraphics, cxLookAndFeels, cxLookAndFeelPainters, cxNavigator, cxLabel;

type
  TfrmSchedule = class(TfrmBasic)
    cxDBTextEdit1: TcxDBTextEdit;
    cxDBTextEdit2: TcxDBTextEdit;
    cxDBTextEdit3: TcxDBTextEdit;
    cxDBTextEdit4: TcxDBTextEdit;
    cxDBTextEdit5: TcxDBTextEdit;
    cxDBTextEdit6: TcxDBTextEdit;
    cxDBTextEdit7: TcxDBTextEdit;
    cxDBTextEdit8: TcxDBTextEdit;
    cxDBTextEdit9: TcxDBTextEdit;
    Label1: TcxLabel;
    Label2: TcxLabel;
    Label3: TcxLabel;
    Label4: TcxLabel;
    Label5: TcxLabel;
    Label6: TcxLabel;
    Label7: TcxLabel;
    Label8: TcxLabel;
    Label9: TcxLabel;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  frmSchedule: TfrmSchedule;

implementation

{$R *.dfm}

uses
  IssueListForms, IssueListConst, IssueListData;


initialization

  FormManager.RegisterForm(ScheduleFormID, TfrmSchedule, 'SCHEDULE');

end.

File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/CBuilder/IssueList/IssueListSchedule.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IssueListSchedule.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "IssueListForm"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxDBEdit"
#pragma link "cxEdit"
#pragma link "cxTextEdit"
#pragma link "cxGraphics"
#pragma link "cxLabel"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxLookAndFeels"
#pragma link "cxNavigator"
#pragma resource "*.dfm"
TfrmSchedule *frmSchedule;
//---------------------------------------------------------------------------
__fastcall TfrmSchedule::TfrmSchedule(TComponent* Owner)
  : TfrmBasic(Owner)
{
}
//---------------------------------------------------------------------------
